mir-ci/mir_ci/test_tests.py

```diff
@@ -3,18 +3,24 @@
 import time
 import pytest
 import asyncio
+import subprocess
+import time
```

Comment:
```suggestion
```

Reply:
Not sure why the reordering? Did anything do that automatically? Can't grok the rules, why it did?